.papr-master-ha.inventory

```diff
@@ -12,7 +12,6 @@ openshift_master_default_subdomain="{{ lookup('env', 'RHCI_ocp_master1_IP') }}.x
 openshift_check_min_host_disk_gb=1.5
 openshift_check_min_host_memory_gb=1.9
 openshift_portal_net=172.30.0.0/16
-openshift_enable_service_catalog=false
```

Comment:
How can we upgrade something that's not installed?

Reply:
oh, correct, PAPR is running on openstack, which won't install service catalog in 3.9